StyleCop.Analyzers/StyleCop.Analyzers.Test/OrderingRules/SA1207UnitTests.cs

```diff
@@ -54,11 +60,11 @@ private static IEnumerable<string> ValidAccessModifiers
             }
         }
 
-        private static IEnumerable<string> InvalidAccessModifiers
+        private static IEnumerable<Tuple<string, string>> InvalidAndFixedAccessModifiers
         {
             get
             {
-                yield return "internal protected";
+                yield return Tuple.Create("internal protected", "protected internal");
```

Comment:
:exclamation: Add the following tests:

| Before | After |
| --- | --- |
| `internal static protected` | `protected static internal` |
| `internal protected abstract` | `protected internal abstract` |
| `/*comment*/internal /* 2 */  protected /*3*/` | `/*comment*/protected /* 2 */  internal /*3*/` |